pkg/libvmi/cloudinit.go

```diff
@@ -99,3 +99,14 @@ func addDiskVolumeWithCloudInitNoCloud(vmi *v1.VirtualMachineInstance, diskName
 func setCloudInitNoCloud(volume *v1.Volume, source *v1.CloudInitNoCloudSource) {
 	volume.VolumeSource = v1.VolumeSource{CloudInitNoCloud: source}
 }
+
+func WithoutCloudInitNoCloud() Option {
```

Comment:
It seems that you add a function into `libvmi` just in order to fix a limitation in a higher-level `libvmifact`. This sounds a bit awkward, maybe I don't understand you.